cobbler/tftpgen.py

```diff
@@ -627,6 +642,10 @@ def write_pxe_file(self, filename, system, profile, distro, arch,
 
         # save file and/or return results, depending on how called.
         buffer = self.templar.render(template_data, metadata, None)
+
+        if "serial" in metadata and metadata["serial"]:
```

Comment:
Maybe we should remove these two lines and should only provide the '$serial' variable which can be added manually in the template files by the users?

Reply:
I think this is fine, let's get this feature in first.